src/app/app.module.ts

```diff
@@ -51,7 +51,7 @@ import {MatSortModule} from '@angular/material/sort';
 import {ProblemSetComponent} from './components/problems/problem-set/problem-set.component';
 import {ProblemViewComponent} from './components/problems/problem-view/problem-view.component';
 import {ProblemEditComponent} from './components/problems/problem-edit/problem-edit.component';
-import {CommonModule} from '@angular/common';
+import {CommonModule, DatePipe} from '@angular/common';
```

Comment:
Unused Import.

Reply:
maybe just undo all the changes in this file.

Reply:
Your choice on the TSLint cause the file reformatted this way.